Doc/whatsnew/3.7.rst

```diff
@@ -283,6 +283,10 @@ New function :func:`os.register_at_fork` allows registering Python callbacks
 to be executed on a process fork.  (Contributed by Antoine Pitrou in
 :issue:`16500`.)
 
+The ``os.stat_float_times()`` function has been removed. It was introduced in
```

Comment:
It should be in other section, "API and Feature Removals".

Reply:
done